pkg/schedule/schedulers/evict_slow_store.go

```diff
@@ -308,7 +350,113 @@ func (s *evictSlowStoreScheduler) IsScheduleAllowed(cluster sche.SchedulerCluste
 // Schedule implements the Scheduler interface.
 func (s *evictSlowStoreScheduler) Schedule(cluster sche.SchedulerCluster, _ bool) ([]*operator.Operator, []plan.Plan) {
 	evictSlowStoreCounter.Inc()
+	if len(s.conf.getNetworkSlowStores()) != 0 || s.conf.getMaxNetworkSlowStore() != 0 {
```

Comment:
Why use `||`?

Reply:
If a slow store already exists and has been detected by PD, and the user adjusts MaxNetworkSlowStore, should we allow the slow store to recover?